website/docs/r/org_vdc_template.html.markdown

```diff
@@ -0,0 +1,157 @@
+  If set, a `edge_gateway` block **must** be present in the VDC Template configuration (see below).
+  * `services_edge_cluster_id` - (Optional) ID of the Edge Cluster that the VDCs instantiated from this template will use for services.
+  Can be obtained with [`vcd_nsxt_edge_cluster` data source](/providers/vmware/vcd/latest/docs/data-sources/nsxt_edge_cluster)
+* `allocation_model` - (Required) Allocation model that the VDCs instantiated from this template will use.
+  Must be one of: `AllocationVApp`, `AllocationPool`, `ReservationPool` or  `Flex`
+* `compute_configuration`: The compute configuration for the VDCs instantiated from this template:
+  * `cpu_allocated` - (Required for `AllocationPool`, `ReservationPool` or `Flex`) The maximum amount of CPU, in MHz, available to the VMs running within the VDC that is instantiated from this template. Minimum is 256MHz
+  * `cpu_limit` - (Required for `AllocationVApp`, `ReservationPool` or `Flex`) The limit amount of CPU, in MHz, of the VDC that is instantiated from this template. Minimum is 256MHz. 0 means unlimited
+  * `cpu_guaranteed` - (Required for `AllocationVApp`, `AllocationPool` or `Flex`) The percentage of the CPU guaranteed to be available to VMs running within the VDC instantiated from this template
+  * `cpu_speed` - (Required for `AllocationVApp`, `AllocationPool` or `Flex`) Specifies the clock frequency, in MHz, for any virtual CPU that is allocated to a VM. Minimum is 256MHz
+  * `memory_allocated` - (Required for `AllocationPool`, `ReservationPool` or `Flex`) The maximum amount of Memory, in MB, available to the VMs running within the VDC that is instantiated from this template
+  * `memory_limit` - (Required for `AllocationVApp`, `ReservationPool` or `Flex`) The limit amount of Memory, in MB, of the VDC that is instantiated from this template. Minimum is 1024MB. 0 means unlimited
+  * `memory_guaranteed` - (Required for `AllocationVApp`, `AllocationPool` or `Flex`) The percentage of the Memory guaranteed to be available to VMs running within the VDC instantiated from this template
+  * `elasticity` - (Required for `Flex`) True if compute capacity can grow or shrink based on demand
+  * `include_vm_memory_overhead` - (Required for `Flex`) True if the instantiated VDC includes memory overhead into its accounting for admission control
+* `storage_profile` - (Required) A block that defines a storage profile that the VDCs instantiated from this template will use. Must be **at least one**, which has the following properties:
+  * `name` - (Required) Name of Provider VDC storage profile to use for the VDCs instantiated from this template
+  * `default` - (Required) True if this is default storage profile for the VDCs instantiated from this template. Only **one** block should have this set to `true`
+  * `limit` - (Required) Storage limit for the VDCs instantiated from this template, in Megabytes. 0 means unlimited
```

Comment:
Nit: in above params we state "in MB", but here "in Megabytes". What about making them consistent?

Reply:
Fixed!